docs/CodeReviewRulesOfThumb.md

```diff
@@ -0,0 +1,76 @@
+  - If possible, add examples of API calls that could be made to test the PR change
+- Tag your PRs with the appropriate labels to help provide context for the release notes (see [Squash Merge Requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md) for more details)
+
+## Make comments actionable
+
+|Instead of This|Try This|Reasoning|
+|---|---|---|
+|“This method is too long”|“This method is quite long, maybe you could extract this part as a sub method”|Provide a suggestion|
+|“I don't understand this”|“It's hard to understand this because I don't see why we need to use a dictionary here instead of a list”|Ask specific questions|
+|“This method should be async”|“This method should be async, otherwise it could cause deadlocks”|Explain why|
+|“Hey, you should remove this line”|“Marking this as a must-fix before merging because this line will break the server/client connection.”|Prioritize your comments|
+|“Maybe try a different name for this variable”|“It would be good to rename this to something like `CurrencyValueSet` to avoid the name `SystemValues`, since the `ValueSet` uri is "http://hl7.org/fhir/ValueSet/currencies" and the `CodeSystem` is "urn:iso:std:iso:4217" ([here](https://www.hl7.org/fhir/valueset-currencies.html) is the relevant part of the spec).”|Help suggest a new name if you think a variable could be renamed|
+
+## Ask questions to gain understanding
+
+- Not all comments are actionable
+- Preface non-actionable comments with "for my understanding"
+- It's important to continue to ask these questions
+
+## Practice healthy communication
+
+|Instead of This|Try This|Reasoning|
+|---|---|---|
+|"You forgot to add method comment"|"This method is missing a comment"|Avoid words like "mine and "yours"|
+|“You should use a constant”|"Here it seems we need a constant because..."|Use the word "we"|
+|"Rename to user_id"|"What do you think of renaming to user_id?"|If fitting, ask questions instead of making demands|
+
+## Be humble and assume the best of the author
+
+- Ask questions before categorizing something as an error
+
+## Move long PR discussions to video calls
+
+- Be open to moving discussions to chat or video calls for clarification
+- Post a resolution comment so other people reading the PR know what happened
+
+## Positive reinforcement is important!
+
+- If you think it is appropriate and would like to add an emoji, the team has expressed that we like them 🙂
+
+## Additional notes
+
+- Consider using [Code Flow](https://www.1eswiki.com/wiki/CodeFlow_integration_with_GitHub_Pull_Requests), a tool that integrates with Github to provide a helpful UI for code reviewing
+- If the same issue occurs in multiple places in one file, add one comment that mentions that this occurs more than once (reconsider for multiple files)
+- It is nice to proceed nitpick comments with “nit”
```

Comment:
>proceed [](start = 16, length = 7)

nit: typo, I think this should be precede?

Reply:
I think I have been using "proceed" instead of "precede" my entire life, oops 😅. Thanks! Fixed in d690eb1.